### PR TITLE
Fix GH issue #5404 (DEV)

### LINKED ISSUE #5404: Missing guidance about the choice of branch for PR submitters
The [CONTRIBUTING](https://github.com/corona-warn-app/cwa-app-android/blob/main/CONTRIBUTING.md) guide and the linked [pull_request_template](https://github.com/corona-warn-app/cwa-app-android/blob/main/.github/pull_request_template.md) do not mention how to choose a branch when submitting a PR.

## Suggestion

Add some guidance about how to choose the most appropriate branch when submitting a PR.

### PR DESCRIPTION
**Pull request by mtwalli**
Fixes #5404

### CODE REVIEW

**Review by Ein-Tim — COMMENTED** [review]
One comment.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -7,28 +7,41 @@ Remove the checklist after fulfilling all the relevant points, and before creati
...
+      e.g.: `Use logger (closes # 41)`.
+    * If this PR comes from a
+      fork,please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
+    * The PR can not contain changes in localization files, e.g. `values-[LANGUAGE-CODE]/strings/*`
+      and `/`
+      or `assets/*` (see [#332](https://github.com/corona-warn-app/cwa-app-android/issues/332) for
+      further information).
+    * Before submitting a PR, please make sure that you target the active `release/*.*.*` branch
+      which can be determined by the active milestone on GitHub
+    * Set `community` label
```
[the remark this answers is not in the document]
Comment (mtwalli) [reply .github/pull_request_template.md]: Updated

**Comment by mtwalli** [comment]
@Ein-Tim , @MikeMcC399  Please let me know if the proposed change is OK from community point of view ?

**Review by Ein-Tim — DISMISSED** [review]
LGTM now @mtwalli!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -7,28 +7,41 @@ Remove the checklist after fulfilling all the relevant points, and before creati
...
-    * Please link to an issue ticket where your code change has been greenlit, otherwise it's unlikely it can be merged.
-    * Use a descriptive title: {task_name} (closes #{issue_number}), e.g.: `Use logger (closes # 41)`.
-    * If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
-    * The PR can not contain changes in localization files, e.g. `values-[LANGUAGE-CODE]/strings/*` and `/` or `assets/*` (see [#332](https://github.com/corona-warn-app/cwa-app-android/issues/332) for further information).
+    * Please link to an issue ticket where your code change has been greenlit, otherwise it's
+      unlikely it can be merged.
+    * Use a descriptive title: {task_name} (closes #{issue_number}),
+      e.g.: `Use logger (closes # 41)`.
+    * If this PR comes from a
+      fork,please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
```
Comment (MikeMcC399) [inline .github/pull_request_template.md:18]:
```suggestion
      fork, please [allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -7,28 +7,41 @@ Remove the checklist after fulfilling all the relevant points, and before creati
...
+    * The PR can not contain changes in localization files, e.g. `values-[LANGUAGE-CODE]/strings/*`
+      and `/`
+      or `assets/*` (see [#332](https://github.com/corona-warn-app/cwa-app-android/issues/332) for
+      further information).
+    * Before submitting a PR, please make sure that you target the active `release/*.*.*` branch
+      which can be determined by the active milestone on GitHub
+    * Include `community` in PR title such as "PR is fixing issue ### (community)"
```
Comment (MikeMcC399) [inline .github/pull_request_template.md:25]:
```suggestion
    * Include `COMMUNITY` in PR title such as "PR fixes issue ### (COMMUNITY)"
```
1. In the past mostly "COMMUNITY" was used, sometimes "Community" but never "community" (in lower case).
2. "is fixing" is a problematic grammar usage. "fixes" is timeless and better.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -7,28 +7,41 @@ Remove the checklist after fulfilling all the relevant points, and before creati
...
+* To _everyone_ :world_map::
+    * Describe your changes in detail, if you changed the UI, screenshots or GIFs would be awesome!
+    * Short step-by-step instructions help the reviewer test your changes, e.g. how to navigate to a
+      new UI element you added.
+    * The PR _won't be reviewed_ if CI is failing or if there are merge conflicts. If CI is still
+      failing mark the PR as a draft and write a little comment on your status.
+    * Provide at least a few unit and/or instrumentation tests.
+    * Use a meaningful branch name. Use either `fix` or `feature` as prefix for your branch,
+      e.g. `fix/prevent-npe-on-device-rotation-issue_123`
+    * Test your changes thoroughly. Only open PRs which you think is ready to be merged. If you
```
Comment (MikeMcC399) [inline .github/pull_request_template.md:41]:
```suggestion
    * Test your changes thoroughly. Only open PRs which you think are ready to be merged. If you
```
grammar

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Added some minor feedback in-line.

**Review by MikeMcC399 — APPROVED** [review]
👍🏻

**Pull request merged**